### ISSUE TITLE
Version 0.17 does not compile on 32 bit Linux

### ISSUE THREAD

**Issue by peti** [haskell-gi/haskell-gi-base#7]
We're seeing the following error:

~~~~
[  104s] src/Data/GI/Base/BasicConversions.hsc:449:16:
[  104s]     Not in scope: type constructor or class ‘Word32’
[  104s]     Perhaps you meant one of these:
[  104s]       ‘Word’ (imported from Prelude), ‘Word8’ (imported from Data.Word),
[  104s]       ‘Word64’ (imported from Data.Word)
~~~~

The complete build log is (temporarily) at https://build.opensuse.org/package/live_build_log/devel:languages:haskell:lts:6/ghc-haskell-gi-base/openSUSE_Tumbleweed/i586.

**Comment by garetxe** [comment]
Thanks for the report. I think that git HEAD should fix this (via commit eeb54b9f5e50bda92fbfc09a6a777decd2d2d574). Could you please check and let me know if it works?

**Comment by peti** [comment]
Yes, with that patch applied the package compiles fine.
